packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -44,6 +45,17 @@ const IModelLoader = React.memo((viewerProps: ModelLoaderProps) => {
     backstageItems,
     loadingComponent,
   } = viewerProps;
+
+  const newUiProviders = useMemo<UiItemsProvider[]>(() => {
+    if (backstageItems && uiProviders) {
```

Comment:
look at this over again;
what happens when uiProviders is undefined or an empty array?

also maybe dont call this "newUiProviders", maybe providers would suffice?